packages/ui/src/components/theme-selector-v2/index.tsx

```diff
@@ -0,0 +1,229 @@
+                  }}
+                  className="hidden"
+                />
+                <div className="h-4 w-4 rounded-full border border-gray-600 flex items-center justify-center">
+                  {contrast === item && <div className="h-2 w-2 rounded-full bg-white" />}
+                </div>
+                <span className="text-gray-300">{item}</span>
+              </label>
+            ))}
+          </div>
+        </div>
+
+        <Separator className="h-px bg-gray-800" />
+
+        {/* Color Adjustment */}
+        <div>
+          <div className="flex flex-col gap-1">
+            <Text className="text-md font-medium">Color Adjustment</Text>
+            <Text className="text-xs text-foreground-3">Adjust colors for different types of color blindness.</Text>
+          </div>
+          <div className="mt-4 flex gap-2">
+            {Object.values(ColorAdjustment).map(item => (
+              <label key={item} className="flex cursor-pointer items-center gap-2">
+                <input
+                  type="radio"
+                  name="color-adjustment"
+                  value={item}
+                  checked={colorAdjustment === item}
+                  onChange={() => {
+                    setColorAdjustment(item)
+                    onChange({ colorAdjustment: item })
+                  }}
+                  className="hidden"
+                />
+                <div className="h-4 w-4 rounded-full border border-gray-600 flex items-center justify-center">
+                  {colorAdjustment === item && <div className="h-2 w-2 rounded-full bg-white" />}
+                </div>
+                <span className="text-gray-300">{item}</span>
+              </label>
+            ))}
+          </div>
+        </div>
+
+        <Separator className="h-px bg-gray-800" />
+
+        {/* Accent Color */}
+        <div>
+          <div className="flex flex-col gap-1">
+            <Text className="text-md font-medium">Accent Color</Text>
+            <Text className="text-xs text-foreground-3">Select your application accent color.</Text>
+          </div>
+          <div className="mt-4 flex gap-2">
+            {Object.values(AccentColor).map(item => (
+              <button
+                key={item}
+                className={cn('h-6 w-6 rounded-full border', accentColor === item ? 'border-white' : 'border-gray-600')}
+                style={{ backgroundColor: item }}
+                onClick={() => {
+                  setAccentColor(item)
+                  onChange({ accentColor: item })
+                }}
+              />
+            ))}
+          </div>
+        </div>
+
+        {/* Buttons */}
+        <Dialog.Footer>
+          <ButtonGroup>
+            <>
+              <Button variant="secondary" onClick={onCancel}>
+                Cancel
+              </Button>
+              <Button onClick={() => onSave({ mode, contrast, colorAdjustment, accentColor })}>Save preferences</Button>
+            </>
```

Comment:
No need for the fragment

```suggestion
            <Button variant="secondary" onClick={onCancel}>
              Cancel
            </Button>
            <Button onClick={() => onSave({ mode, contrast, colorAdjustment, accentColor })}>
              Save preferences
            </Button>
```

Reply:
Done, removed it.